examples/rust-gcoap/Cargo.lock

```diff
@@ -0,0 +1,896 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
```

Comment:
should this be committed then?

Reply:
Yes: This is what ensures that things don't break just because any third party in the dependency chain declares an update to be compatible when it actually isn't.

Think of this file as the hashes of several pkgs that are all managed in here rolled into one. Like with packages, they'll be more or less regularly updated, but manually. (Typically, all a Cargo.lock file is updated in one go, so whenever riot-sys or -wrappers grow a feature needed for an example, everything else gets updated too).